packages/engine/README.md

```diff
@@ -219,13 +217,83 @@ There are overrides ([CLI Arguments and Options](#cli-arguments-and-options)) fo
 
 By default, the engine outputs a serialized snapshot of Agent state every step.
 
-During the run, the output may be buffered into the `./parts` folder in multiple files. These files are not necessarily valid JSON as the resultant state blob that appears within `json_state.json` is split up (hence `part`) for buffering purposes. 
+During the run, the output may be buffered into the `./parts` folder in multiple files. These files are not necessarily valid JSON as the resultant state blob that appears within `json_state.json` is split up (hence `part`) for buffering purposes.
 
 #### Analysis [`analysis_outputs.json`]
 > **WIP** - This feature is currently unstable
 
 [hCore] currently provides functionality where simulations can apply custom analysis on user-defined metrics. The functionality has been ported across to this codebase in the [analysis package](./src/simulation/package/output/packages/analysis), however development is planned to stabilise it. As such, this functionality is neither tested, nor considered supported.
 
+### Behavior keys
```

Comment:
I think this doesn't quite make sense as its own section. We should probably add a `### Simulation Inputs` section similar to `Simulation Outputs` and start describing the simulation project folder structure.

To avoid blowing up the scope of the PR though, we can avoid fleshing out all of it and just stick another WIP disclaimer